examples/CivisML_parallel_training.ipynb

```diff
@@ -220,16 +234,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Another thing I'm often doing is building lots of models with tweaked hyperparameters, to see what fits best.\n",
+    "Another thing I'm often doing is building lots of models with tweaked hyperparameters, to see what fits best. Often the choice of hyperparameter has a bigger impact on the outcome than the model itself, so it was important to use to make hyperparameter searches as easy as possible.  \n",
```

Comment:
"so it was important to use to make hyperparameter searches as easy as possible."
What is this trying to say? Should it be
"so CivisML makes hyperparameter searches as easy as possible."?